boards-ci-2023.06.yaml

```diff
@@ -0,0 +1,29 @@
+---
```

Comment:
why using a version number in the settings file? settings files shouldn't be versioned.

Reply:
This file is used for CI.